en/includes/guides/authentication/conditional-auth/configure-conditional-auth.md

```diff
@@ -106,4 +106,14 @@ To delete an existing secret:
 
 5. Select the checkbox and confirm your action.
 
-{% endif %}
\ No newline at end of file
+### Using Secrets in Scripts
```

Comment:
Other headings seems to be in sentence case. So update similarly.

Reply:
And better heading is termed as 'Use secret in the script'

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4773/commits/9acc22f1d66873e2c67cb71939862187674bece9